lib/kender/commands/cypress_rails.rb

```diff
@@ -0,0 +1,15 @@
+module Kender
+  class CypressRails < Command
+
+    # Kender might be in some projects which use cypress-rails but
+    # do not have this rake task.
+    def available?
+      in_gemfile?('cypress-rails') && Rake::Task.task_defined?('cypress_extended:enhanced_run')
+    end
+
+    def command
+      'rake cypress_extended:enhanced_run'
```

Comment:
All of the other commands use `bundle exec` and that is typically for most setups so I think it is safer to include it here, or does it not work for you with `bundle exec`?
```suggestion
      'bundle exec rake cypress_extended:enhanced_run'
```